### PR TITLE
feat/#225 upgrade setting password

### PR DESCRIPTION
**Pull request by Sintry1**
### This pull request closes:

✔️ Closes Jira Issue: [TC-](https://treecreate.atlassian.net/browse/TC-)

### Which service(s) does this issue affect:

- [ ] 🛰️ API
- [x] 🛒 webstore
- [ ] 🎓 admin-page

### Types of changes

- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
- [x] 🧱 New feature (a non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🤖 CI/CD (a change to the CI/CD pipeline)
- [ ] ⚗️ Refactoring (an update to some already existing code)
- [x] 💄 Style (Markup, formatting, CSS)

### Documentation Updates

- [x] 📂 No need for updates
- [ ] 📁 Requires an update ( _not done within this issue_ )
- [ ] 🗃️ Has been updated ( _done within this issue_ )

### Testing checklist

- [x] 💪 Manual tests
- [ ] 🔧 Automatic tests

#### Browser compatibility - _**[**Frontend only**]**_

- [ ] Tested on Safari
- [x] Tested on Chrome
- [ ] Tested on Mozilla Firefox

### What changes have been done within this issue?

- Created css classes for text styling
- Created methods to check each condition
- Added method calls to signup.component.html 

### Screenshots or example usage

image
image

### CODE REVIEW

File: apps/webstore/src/app/pages/auth/signup/signup.component.html [hunk 1]
```diff
@@ -56,8 +56,11 @@ <h1 class="page-title" i18n="@@signup">Sign up</h1>
             </button>
         </div>
         <div class="info-message" i18n="@@info-password">
```
Comment (Teodor25) [inline apps/webstore/src/app/pages/auth/signup/signup.component.html]: FYI: This part needs to be translated properly. So this issue cannot be approved before i do translations.

Comment (Sintry1) [reply apps/webstore/src/app/pages/auth/signup/signup.component.html]: I'm also doing the reset password page, so it's not quite complete yet.

Comment (Sintry1) [reply apps/webstore/src/app/pages/auth/signup/signup.component.html]: I've done the reset password page now and requested a re-review. 
[https://gyazo.com/7c53d67d660e5a17a5e756deb3721ed6](url)
[https://gyazo.com/6d03515ce536f7955233b1da9980a248](url)

Do you want the styling changed from how this looks or are you happy to go ahead with translations?

Comment (Teodor25) [reply apps/webstore/src/app/pages/auth/signup/signup.component.html]: I cant see the screenshots. Just add them in the description ;)

Comment (Sintry1) [reply apps/webstore/src/app/pages/auth/signup/signup.component.html:58]: image
image

**Review by Teodor25 — COMMENTED** [review]
Looks good, now it just needs to pass the tests and have localization (ill do localization)

File: apps/webstore/src/app/pages/auth/signup/signup.component.html (hunk 1 above)
Comment (Sintry1) [reply apps/webstore/src/app/pages/auth/signup/signup.component.html]: Resolved : [e3c6d7d](https://github.com/treecreate/webstore/pull/377/commits/e3c6d7d1747d804aa52db8d9ba6b1e69a4c0fdc3)

**Review by Teodor25 — APPROVED** [review]

**Pull request merged**